Makefile.in

```diff
@@ -9,6 +9,7 @@ COMPAT_VERSION	= $(VERSION_MAJOR)
 
 CC		= @CC@
 CFLAGS		= -c @CFLAGS@ @DEFS@ -D_REENTRANT
+#CFLAGS += -DDEBUG
```

Comment:
This seems like it should be indented with two tabs.

Reply:
fixed